readthedocs/projects/migrations/0031_add_modified_date_importedfile.py

```diff
@@ -8,7 +8,7 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('projects', '0029_add_additional_languages'),
+        ('projects', '0030_add_modified_date_importedfile'),
```

Comment:
Ups, I approved, but this should be `0030_change-max-length-project`

Reply:
ha, indeed _sigh_

Reply:
Good thing CI caught it :D